backend/letter/models.py

```diff
@@ -51,7 +51,35 @@ def __str__(self):
         return f"{self.name} ({self.material}), gifted by {gifter_name}"
 
 
-class Letter(models.Model):
+class GiftDescription(SourceDescription, GiftBase, models.Model):
+    """
+    A description of a gift in a source.
+    """
+
+    target = models.ForeignKey(
+        to=GiftBase,
+        on_delete=models.CASCADE,
+        related_name="source_descriptions",
+        null=True,
+        blank=True,
+    )
+
+    def __str__(self):
+        """
+        Inherits the __str__ method from the base model and adds the source to it.
+        """
+        return f"{super().__str__()} (described in {self.source})"
+
+
+class Gift(Historical, GiftBase, models.Model):
```

Comment:
Same thing as for actions: I think we can start out without a `Historical` model for gifts and letters, though we can add that layer of analysis when needed.